cnf-tests/testsuites/e2esuite/security/tuning_sriov.go

```diff
@@ -50,7 +52,18 @@ var _ = Describe("[sriov] Tuning CNI integration", func() {
 			networks.CleanSriov(sriovclient, SriovTestNamespace)
 			sysctls, err := networks.SysctlConfig(map[string]string{fmt.Sprintf(Sysctl, "IFNAME"): "1"})
 			Expect(err).ToNot(HaveOccurred())
-			networks.CreateSriovPolicyAndNetwork(sriovclient, namespaces.SRIOVOperator, "test-network", "testresource", sysctls)
+			networks.CreateSriovPolicyAndNetwork(sriovclient, namespaces.SRIOVOperator, "test-network", "testresource", fmt.Sprintf("{%s}", sysctls))
```

Comment:
general question why we create the network-attachment on the sriov operator namespace?

Reply:
Looks like an issue of the original test.
Let's fix it in a follow up PR